torch/csrc/distributed/c10d/UCCTracing.cpp

```diff
@@ -34,15 +34,15 @@ void ProcessGroupUCCLogger::flushComms(int rank, int world_size) {
         "_", (1 + ltm->tm_mon), "_", ltm->tm_mday, "_", (1900 + ltm->tm_year));
   }
 
-  std::filesystem::path fullpath = std::filesystem::path("/tmp") / dirname;
+  c10::filesystem::path fullpath = c10::filesystem::path("/tmp") / dirname;
```

Comment:
FUTURE PR, this should really use $TMPDIR env vars for filesystems that don't have tmp. It's sucks that this is hardcoded